server/services/store/sqlstore/team.go

```diff
@@ -0,0 +1,217 @@
+package sqlstore
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/mattermost/focalboard/server/model"
+	"github.com/mattermost/focalboard/server/utils"
+
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+
+	sq "github.com/Masterminds/squirrel"
+)
+
+var (
+	errUnsupportedOperation = errors.New("unsupported operation")
+	teamFields              = []string{
+		"id",
+		"signup_token",
+		"COALESCE(settings, '{}')",
+		"modified_by",
+		"update_at",
+	}
+)
+
+func (s *SQLStore) upsertTeamSignupToken(db sq.BaseRunner, team model.Team) error {
+	now := utils.GetMillis()
+
+	query := s.getQueryBuilder(db).
+		Insert(s.tablePrefix+"teams").
+		Columns(
+			"id",
+			"signup_token",
+			"modified_by",
+			"update_at",
+		).
+		Values(
+			team.ID,
+			team.SignupToken,
+			team.ModifiedBy,
+			now,
+		)
+	if s.dbType == mysqlDBType {
+		query = query.Suffix("ON DUPLICATE KEY UPDATE signup_token = ?, modified_by = ?, update_at = ?",
+			team.SignupToken, team.ModifiedBy, now)
+	} else {
```

Comment:
I know team store methods shouldn't be run with sqlite, however I think we should explicitly look for postgres in the `else`, and error on any other db type.

Reply:
In this case the method is related to the signup token, which runs on Personal Server instances, so it should run with SQLite, and the `else` syntax is the one SQLite supports. This is covered on the upsert store tests, for example